apps/desktop/src/lib/forge/github/githubPrService.ts

```diff
@@ -94,7 +97,12 @@ export class GitHubPrService implements ForgePrService {
 	}
 
 	prMonitor(prNumber: number): GitHubPrMonitor {
-		return new GitHubPrMonitor(this, prNumber);
+		if (this.monitors.has(prNumber)) {
+			return this.monitors.get(prNumber)!;
+		}
+		const monitor = new GitHubPrMonitor(this, prNumber);
+		this.monitors.set(prNumber, monitor);
+		return monitor;
```

Comment:
We have no mechanism with which to clean up this the map, which means they won't get garbage collected.. Let's discuss over video? I actually just removed the duplicated pr monitor yesterday.